enforcerlbp/enforcerWeb/Controllers/AccountController.cs

```diff
@@ -257,7 +259,106 @@ private async Task<ResponseModel> LogUserIn(LoginDTO model)
+                //return an error if it is
+                return BadRequest(new ResponseModel { Response = "Supply user email", Status = false });
+
+            try
+            {
+                //check if the user exists
+                var user = await _userManager.FindByEmailAsync(email);
+
+                if (user == null)
+                    //if not return an error
+                    return BadRequest(new ResponseModel { Response = "Oops seems you do not have an account with us. You can easily register now", Status = false });
+
+                //generate a reset token from OTPGenerator class
+                 var Token = OTPGenerator.GetOTPHash("eagi");
+                 
+                //send  the user an email template containing a reset link and OTP
+                await _mediatR.Send(new EmailSenderCommand
+                {
+                    Contacts = new List<ContactsModel>()
+                {
+                    new ContactsModel
+                    {
+                        Email = "afeexclusive@gmail.com",
+                        Name = "Ayokunle"
+                    },
+                    new ContactsModel
+                    {
+                        Email = "adeoyetemitayo99@gmail.com",
+                        Name = "Tayo"
+                    }
+                },
+                    EmailDisplayName = "Health Enforcer",
+                    Message = Token.otp,
+                    Subject = "Password Reset OTP",
+                    Category = "",
+
+
+                }) ;
+
+                return Ok(new ResponseModel { Response = "One Time Password sent successfully", Status = true });
+            }
+            catch (Exception ex)
+            {
+
+                return Ok(new ResponseModel { Response = ex.InnerException.Message ?? ex.Message, Status = true });
+            }
+        }
+
+        [HttpPost]
+        [Route("passwordreset")]
+        public async Task<IActionResult> ResetPassword(PasswordResetDTO model)
+        {
+            try
+            {
+                // Find the user by email
+                var user = await _userManager.FindByEmailAsync(model.Email);
+
 
+                if (user == null)
+                {
+                    return NotFound();
+                }
+
+                if (user.UserName != user.Email)
+                {
+                    user.UserName = user.Email;
+                    user.NormalizedUserName = user.Email.ToUpper();
+                    await _userManager.UpdateAsync(user);
+                }
+
+
+
+                // reset the user password
+                var token = await _userManager.GeneratePasswordResetTokenAsync(user);
+                var result = await _userManager.ResetPasswordAsync(user, token, model.Password);
+                if (result.Succeeded)
+                {
+                    user.PhoneNumberConfirmed = false;
+                    await _userManager.UpdateAsync(user);
```

Comment:
@AdelabuAderonke Kindly remove this code `user.PhoneNumberConfirmed = false;
                    await _userManager.UpdateAsync(user);` the change can be committed to your next push. Thanks